fitbenchmarking/results_processing/performance_profiler.py

```diff
@@ -436,10 +582,10 @@ def create_graph(self, x_axis_scale, solvers):
                     mode='lines',
                     line={
                         "shape": 'hv',
-                        "dash": linestyles[(i % len(linestyles))],
-                        "color": colors[(i % len(colors))]
+                        "dash": self.current_styles[solver]['linestyle'],
+                        "color": self.current_styles[solver]['linecolor'],
```

Comment:
This is then a little less obvious but I think it's neater in general?
```suggestion
                        "dash": self.current_styles[solver][0],
                        "color": self.current_styles[solver][1],
```